api/utils.py

```diff
@@ -302,7 +303,7 @@ async def get_one(self, model_id: int, request: Request):
         except HTTPException:
             if self.get_one_auth:
                 raise
-            user = None
+            user = None  # pragma: no cover
```

Comment:
This line is covered when accessing public stores

Reply:
thanks. I will add it